lib/mongoid/fields.rb

```diff
@@ -14,27 +15,6 @@ module Fields
     StringifiedSymbol = Mongoid::StringifiedSymbol
     Boolean = Mongoid::Boolean
 
-    # For fields defined with symbols use the correct class.
-    TYPE_MAPPINGS = {
```

Comment:
This is used in client code, e.g. https://github.com/shoplineapp/mongoid-client-side-encryption/blob/master/lib/mongoid_client_side_encryption/models/field.rb#L6

Reply:
I've now deprecated TYPE_MAPPINGS. They will need to migrate at some point in the future.